packages/manager/apps/ips/src/pages/actions/exportIpToCsv/exportIpToCsv.page.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
+import { NAMESPACES } from '@ovh-ux/manager-common-translations';
+import { OdsText } from '@ovhcloud/ods-components/react';
+import { Modal } from '@ovh-ux/manager-react-components';
+import Loading from '@/components/Loading/Loading';
+import { TRANSLATION_NAMESPACES } from '@/utils';
+import { useExportToCsv } from '@/data/hooks';
+
+export default function ExportIpToCsv() {
+  const { isCSVLoading, handleExportToCsv } = useExportToCsv();
+  const { t } = useTranslation([
+    TRANSLATION_NAMESPACES.exportIpToCsv,
+    NAMESPACES.ACTIONS,
+  ]);
+  const navigate = useNavigate();
+
+  const closeModal = () => {
+    navigate('..');
+  };
+
+  const confirm = async () => {
+    await handleExportToCsv();
+    closeModal();
+  };
+
+  return (
+    <Modal
+      isOpen
+      onDismiss={!isCSVLoading ? closeModal : undefined}
```

Comment:
```suggestion
      onDismiss={closeModal}
```

I don't think we should disable the modal closing when it's loading